docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+h2o_temperature,location=santa_monica,state=CA surface_degrees=63.6,bottom_degrees=49.2 1600756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=55.1,bottom_degrees=51.3 1568756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=50.2,bottom_degrees=50.9 1600756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=55.8,bottom_degrees=40.2 1568756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=54.7,bottom_degrees=40.1 1600756160
+```
+
+| location | state | surface_degrees | bottom_degrees | time |
+| --- | --- | --- | --- | --- |
+| coyote_creek | CA | 55.1 | 51.3 | 1568756160 |
+| coyote_creek | CA | 50.2 | 50.9 | 1600756160 |
+| puget_sound | WA | 55.8 | 40.2  | 1568756160 |
+| puget_sound | WA | 54.7 | 40.1  | 1600756160 |
+| santa_monica | CA | 65.2 | 50.4 | 1568756160 |
+| santa_monica | CA | 63.6 | 49.2 | 1600756160 |
+
+Note that the measurement name doesn't appear anywhere. That's because the table itself represents data from the 
+`h2o_temperature` measurement. Later we'll see how to work with multiple measurements (i.e. tables). Note that the 
+underlying time series would be represented by the combination of `location`, `state`, and the specific field. 
+Results are ordered by default by the columns of the tags and their values and then by time. So we can see that 
+`coyote_creek`, `CA` rows are together in time ascending order.
+
+We'll show later how to specify to the graphing engine what represents an individual time series that you want 
+visualized.
+
+## Basic Queries
+This section will explore some basic queries in Flux to highlight the syntax and a few of the basic concepts of
+the language. For these examples, we'll assume a dataset that has a schema like the one seen above.
+
+### Introduction to basics and getting the last value
+In this example, we'll get the last value of `surface_degrees` for every series in the `h2o_temperature` 
+measurement in the `my_data` bucket. Flux has a shorthand syntax for the common operation of selecting data from 
+a bucket with given criteria for a given time range. Let's look at that first for this example:
+
+```
+@my_data.h2o_temperature{surface_degrees, time}
+```
+
+The `@` symbol at the beginning indicates that we're using the shorthand syntax and that the next set of characters
+will identify the bucket name. After that we have a `.` separator, followed by the measurement name of 
+`h2o_temperature`. Immediately following that we have the **Column Filter Predicate** syntax (the curly braces
+`{` and `}` and what's inside of them) to indicate which columns we want in the result. The identifiers can
+also be wrapped in double quotes, which you'll need to use if they contain spaces:
+
+```
+@"my_data"."h2o_temperature"{"surface_degrees", "time"}
+```
+
+Given the previous data set, this query would produce a result like this: 
+
+| surface_degrees | time |
```

Comment:
This result confuses me. I think it would be useful to explain before showing this that if you just add the column names to the filter predicate, it just returns that last value. It would make this less confusing IMO.